app/helpers/event_front_end.rb

```diff
@@ -0,0 +1,5 @@
+module EventFrontEnd
+  def name_and_date
```

Comment:
This is definitely what is being returned here... but maybe something like "display_name" might be more representative of what this method is doing. That way if we later decide to add the event organizer name to this values, we wont have a method called `name_and_date` that gives you back: "name and date and organizer"